tools/interpolation/run_idw_interpolation.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="interpolation_run_idw_interpolation" name="Run IDW interpolation" version="1.0" profile="22.05">
+    <description>based on a .csv and .geojson file</description>
```

Comment:
Not sure if the datatype is meaningful here. Wouldn't it be better to say what the content is?

What is IDW? Should this be spelled out?